packages/reactive-element/src/reactive-element.ts

```diff
@@ -728,21 +729,22 @@ export abstract class ReactiveElement
     key: string | symbol,
```

Comment:
Does the comment on this method need to be updated? Specifically the part: `If no descriptor is returned, the property will not become an accessor.`

Reply:
That comment is still true because of the ` if (descriptor !== undefined)` check in createProperty